Assets/Scripts/CombatController.cs

```diff
@@ -161,6 +163,8 @@ public void EndAction(int spentActionPoints)
         isActing = false;
```

Comment:
On line 161, we don't need to call AssignCurrentTile() since it's called from inside FindSelectableTiles()

Reply:
Yeah, this was to cover the case where there are no action points left - but a better solution is to make the call from inside FindSelectableTiles not dependent on there being action points left.